shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -576,6 +576,8 @@ - (void)testFlutterPlatformViewControllerSubmitFrameWithoutFlutterViewNotCrashin
                                /*raster=*/thread_task_runner,
                                /*ui=*/thread_task_runner,
                                /*io=*/thread_task_runner);
+  std::unique_ptr<flutter::IOSContext> iosContext =
+      flutter::IOSContext::Create(flutter::IOSRenderingAPI::kSoftware);
```

Comment:
No need to create this on the heap, right?  You can just create this on the stack.